api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/WorksQuery.scala

```diff
@@ -34,17 +38,25 @@ case object WorksMultiMatcher {
             (Some(1000), "data.imageData.id.otherIdentifiers.value"),
           ).map(f => FieldWithOptionalBoost(f._2, f._1.map(_.toDouble)))
         ),
-        prefixQuery("data.title.keyword", q).boost(1000),
         MultiMatchQuery(
           q,
-          `type` = Some(CROSS_FIELDS),
-          operator = Some(AND),
+          `type` = Some(BEST_FIELDS),
+          operator = Some(OR),
```

Comment:
Almost 100% sure we don't want the `OR` query, especially boosted so highly.

e.g. `old and new london`

matches `old OR and OR new OR london` and boosts appropriately.